### PR TITLE
VIP-762: Prevent users from viewing feeds from other jurisdictions

### PR DESCRIPTION
**Pull request by alice-dw**
Tested on /xml/overview-summary endpoint:
- `super-admin` can access all feeds
- guest user with FIPS `50` can access Vermont feeds
- guest user with FIPS `51` cannot access Vermont feeds
- guest user with FIPS `50` and `51` can access Vermont feeds

### CODE REVIEW

File: authentication/services.js [hunk 1]
```diff
@@ -21,6 +24,35 @@ const checkJwt = jwt({
...
+  })();
+
+  conn.query(function(client, _) {
+    client.query(query, params, function(err, result) {
+      if (err) {
+        logger.error("Error querying for feeds");
+        logger.error(err);
+        res.writeHead(500, {'Content-Type': 'application/text'});
+        res.write("Server Error");
+        res.end();
```
Comment (alice-dw) [inline authentication/services.js:44]: this error handling is generic, it's done in most callbacks in this repo

File: authentication/services.js [hunk 2]
```diff
@@ -21,6 +24,35 @@ const checkJwt = jwt({
...
+      } else {
+        requestedFeeds = result.rows.filter(feed => feed.public_id == req.params.feedid);
```
Comment (alice-dw) [inline authentication/services.js:46]: this is the main logic

**Review by alice-dw — COMMENTED** [review]
see adjacent pr, only in staging: https://github.com/votinginfoproject/Metis/pull/534

i chose not to use the existing getFeeds functionality because this call doesn't really follow the simpleQueryResponder pattern since we need to use the result of the query before sending it in the response

File: pg/gets.js [hunk 3]
```diff
@@ -16,7 +16,6 @@ var overviewTableRow = function(row, type, dbTable, link) {
 module.exports = {
   // Functions below return arrays for the various queries with the requirement of an ID.
   getFeeds: function(req, res) {
-		var roles = auth.getUserRoles(req);
```
Comment (alice-dw) [inline pg/gets.js:19]: random unused variable

**Review by dwxander — APPROVED** [review]
🔒

**Review by danny-pease — APPROVED** [review]
👾

**Pull request merged**